ledger-tool/src/main.rs

```diff
@@ -381,7 +383,18 @@ fn output_account(
     println!("  rent_epoch: {}", account.rent_epoch());
     println!("  data_len: {}", account.data().len());
     if print_account_data {
-        println!("  data: '{}'", bs58::encode(account.data()).into_string());
+        if encoding == UiAccountEncoding::Base58 {
+            println!(
+                "  data: '[\"{}\",\"base58\"]'",
+                bs58::encode(account.data()).into_string()
+            );
```

Comment:
```suggestion
            println!(" data: '{}'", bs58::encode(account.data()).into_string());
            println!("  encoding: base58);
```
This is to preserve backward compatibility, in case someone is parsing the `data` field programmatically.
Can you please implement this output format for the else case below?

Reply:
I have applied the formating to else case